ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/servers.md

```diff
@@ -14,6 +14,7 @@ port_profiles:
   < port_profile_1 >:
     speed: < interface_speed | forced interface_speed | auto interface_speed >
     mode: < access | dot1q-tunnel | trunk >
+    l2_mtu: < l2_mtu - if defined this profile should be used only for 7050 platform!>
```

Comment:
Change to `if defined this profile should only be used for platforms supporting the "l2 mtu" CLI`